.github/actions/after-e2e/action.yml

```diff
@@ -0,0 +1,24 @@
+name: "After e2e"
+description: "After e2e"
+
+inputs:
+  id:
+    description: 'test suite id'
+    required: true
+    type: number
+
+
+runs:
+  using: "composite"
+  steps:
+    - name: Remove storage dir
+      shell: bash
+      run: |
+        storage_dir="./storage-state/AdminUserState.json"
```

Comment:
you can declare variables with:

```
env:
  storage_dir: "./storage-state/AdminUserState.json"
```

given it's a file not a dir I will adjust also the name

Reply:
Changed